src/towncrier/test/test_project.py

```diff
@@ -39,6 +39,46 @@ def test_tuple(self):
         version = get_version(temp, "mytestproja")
         self.assertEqual(version, "1.3.12")
 
+    def test_import_fails(self):
+        """
+        The project cannot be found.
+        """
+        if sys.version_info >= (3, 6):
+            expected_exception = ModuleNotFoundError
+        else:
+            expected_exception = ImportError
+
+        with self.assertRaises(expected_exception):
+            get_version(".", "mytestproj_import_fails")
+
+    def test_already_installed_import(self):
+        """
+        An already installed package will be checked before cwd-found packages.
+        """
+        project_name = "mytestproj_already_installed_import"
+
+        temp = self.mktemp()
+        os.makedirs(temp)
+        os.makedirs(os.path.join(temp, project_name))
+
+        with open(os.path.join(temp, project_name, "__init__.py"), "w") as f:
+            f.write("__version__ = (1, 3, 12)")
+
+        sys_path_temp = self.mktemp()
+        os.makedirs(sys_path_temp)
+        os.makedirs(os.path.join(sys_path_temp, project_name))
+
+        with open(os.path.join(sys_path_temp, project_name, "__init__.py"), "w") as f:
+            f.write("__version__ = (1, 3, 13)")
+
+        sys.path.insert(0, sys_path_temp)
```

Comment:
Using `addCleanup` should be more idiomatic for cleanup code.

```suggestion
        sys.path.insert(0, sys_path_temp)
        self.addCleanup(sys.path.pop, 0)
```